### ISSUE TITLE
State-of-the-art Deep Learning Methods Example

### ISSUE THREAD

**Issue by robintibor** [braindecode/braindecode#375]
One could have a look at the methods here, and test which ones improve EEG decoding, e.g., on High-Gamma Dataset/BCI Competition IV 2a/TUH Abnormal
@Div12345 
https://pytorch.org/blog/how-to-train-state-of-the-art-models-using-torchvision-latest-primitives/

**Comment by bruAristimunha** [comment]
Is it better to make a list of possible architectures to implement in braindecode? @Div12345, do you have another suggestion?

**Comment by bruAristimunha** [comment]
Do you have some models in mind @robintibor and @agramfort? I am interested in this task, maybe could be a nice task to build the braindecode paper. 

My list:

- [LDMA](https://github.com/MiaoZhengQing/LMDA-Code)
- All the models inside the [BioT](https://github.com/ycq091044/BIOT)
- [TFC](https://github.com/mims-harvard/TFC-pretraining)
- [MAEEG](https://openreview.net/pdf?id=kttuLV59ZuJ)
- [Dist-dcrnn](https://openreview.net/forum?id=k9bx1EfHI_-)
- [EEGClip](https://github.com/tidiane-camaret/EEGClip)
- [Bendr](https://github.com/SPOClab-ca/BENDR)
- [XSleepNet](https://github.com/pquochuy/xsleepnet)
- [EEGGan](https://github.com/AutoResearch/EEG-GAN)
- [EEGViT](https://github.com/ruiqiRichard/EEGViT)
- [Embedding EEG](https://github.com/PierreGtch/embedding_eeg_2022)

**Comment by robintibor** [comment]
Interesting to me could be:
* augmentations, especially channel dropout
* random erasing
* label smoothing
* mipup/cutmix
* EMA or SWA (https://github.com/braindecode/braindecode/issues/239)

**Comment by robintibor** [comment]
also an EEG version of TrivialAugment could be interesting. That would mean selecting a subset of augmentations, creating a discrete set of strengths per augmentation and selecting a random augmentation with a random strength for each example each time. I guess we would need to ensure that all the augmentations are likely to preserve the class label.

**Comment by bruAristimunha** [comment]
The neurips papers that I found interesting and maybe linked with braindecode.

https://neurips.cc/virtual/2023/poster/70774
https://neurips.cc/virtual/2023/poster/71605
https://neurips.cc/virtual/2023/poster/71107
https://neurips.cc/virtual/2023/poster/70217
https://neurips.cc/virtual/2023/poster/72228
https://neurips.cc/virtual/2023/poster/72383
https://neurips.cc/virtual/2023/poster/71117
https://neurips.cc/virtual/2023/poster/72408

**Comment by robintibor** [comment]
wow super interesting thanks @bruAristimunha

**Comment by bruAristimunha** [comment]
One more: https://github.com/duanyiqun/DeWave

**Comment by bruAristimunha** [comment]
Community request to implement the models from @kingjr's team.

1) brainmagick;

Some re-implemenation code here: 
https://github.com/sccn/deep-channel-harmonization

**Comment by bruAristimunha** [comment]
https://arxiv.org/pdf/2502.01678
- MedFormer;
- EEGPY
